test/modules/TestResultCoordinator/Reports/DirectMethod/Connectivity/DirectMethodConnectivityReport.cs

```diff
@@ -68,7 +68,28 @@ class DirectMethodConnectivityReport : TestResultReportBase
         public override string Title => this.ReceiverSource.HasValue ?
             $"DirectMethod Connectivity Report for [{this.SenderSource}] and [{this.ReceiverSource.OrDefault()}] ({this.ResultType})" : $"DirectMethod Report for [{this.SenderSource}] ({this.ResultType})";
 
-        public override bool IsPassed =>
-            this.MismatchFailure == 0 && this.NetworkOffFailure == 0 && this.NetworkOnFailure == 0 && (this.NetworkOnSuccess + this.NetworkOffSuccess + this.NetworkOnToleratedSuccess + this.NetworkOffToleratedSuccess > 0);
+        public override bool IsPassed => this.IsPassedHelper();
+
+        public bool IsPassedHelper()
```

Comment:
As complicate as Connecitivity is... we should definitely update the document on why we need to do this (and how can we undo it once the test is more stable).

Reply:
Adding the logic is okay, but can we add it to the property itself? Or at least make the method private?